tests/client/disco/components/TestInfoDialog.js

```diff
@@ -88,15 +90,15 @@ describe('Clicking outside <InfoDialog />', () => {
           <I18nProvider i18n={getFakeI18nInst()}>
             <div>
               {getInfoDialog()}
-              <div ref="outsideComponent" onClick={(e) => e.stopPropagation()} />
+              <div id="outside-component" onClick={(e) => e.stopPropagation()} />
             </div>
           </I18nProvider>
         );
       }
     }
 
-    const rootComponent = ReactDOM.render(<FakeContainer />, mountNode);
-    const outsideNode = rootComponent.refs.outsideComponent;
```

Comment:
I feel we do this in other places. Should this be using `renderIntoDocument` instead? Does it need to render into a specific element?

If it does need to render into a certain element this could use the third callback argument.

```js
function renderIntoNode(tree, mountNode) {
  return new Promise((resolve) => ReactDOM.render(tree, mountNode, resolve));
}

return renderIntoNode(<FakeContainer />, mountNode)
  .then((rootComponent) => {
    const outsideComponent = rootComponent.refs.outsideComponent;
    // ...
  });